### PR TITLE
Update cloudhub-networking-guide.adoc

### PR DESCRIPTION
**Pull request by jmjdamen**
url fix

### CODE REVIEW

**Comment by hannanelson** [comment]
@sjsheridan - please confirm.

**Review by sjsheridan — CHANGES_REQUESTED** [review]
This change is incorrect. The change would be right for the EU control plane, but that is not covered in this section. The original URLs are correct for the US control plane, which is the example being given here.

Suggest calling out that this is for US instead. I.e.

> Deploying to a region also affects your internal and external worker DNS address. For example, if you deploy in Ireland in the US control plane, the DNS records for internal and external IP addresses are `mule-worker-myapp.ir-e1.cloudhub.io` and `mule-worker-internal-myapp.ir-e1.cloudhub.io`.

**Review by sjsheridan — APPROVED** [review]

**Review by fermujica — APPROVED** [review]

File: modules/ROOT/pages/cloudhub-networking-guide.adoc [hunk 1]
```diff
@@ -133,7 +133,7 @@ supports some of the same regions that the U.S. control plane supports, the DNS
 are different. For more on the EU control plane, see
 xref:eu-control-plane::index.adoc[About the EU Control Plane].
 
-Deploying to a region also affects your internal and external worker DNS address. For example, if you deploy in Ireland, the DNS records for internal and external IP addresses are `mule-worker-myapp.ir-e1.cloudhub.io` and `mule-worker-internal-myapp.ir-e1.cloudhub.io`.
+Deploying to a region also affects your internal and external worker DNS address. For example, if you deploy in Ireland in the US control plane, the DNS records for internal and external IP addresses are `mule-worker-myapp.ir-e1.cloudhub.io` and `mule-worker-internal-myapp.ir-e1.cloudhub.io`.
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/cloudhub-networking-guide.adoc]: if you deploy **in** Ireland **in** the US control plane > This sentence is creating a bit of confusion for me, for example the first part means the user deploys to Ireland or the user is based in Ireland while doing the deployment? 

If possible, I'd prefer to replace "in" with another word that specifically describes the scenario, for example a suggestion could be (assuming what I say is technically correct):

"For example, if you deploy **to** Ireland **using** the US control plane..."

**Review by Cristian-Venticinque — APPROVED** [review]
Please check comment before merging

File: modules/ROOT/pages/cloudhub-networking-guide.adoc (hunk 1 above)
Comment (sjsheridan) [reply modules/ROOT/pages/cloudhub-networking-guide.adoc]: "if you are using the US control plane, and deploy to the Ireland region..."

**Pull request merged**